### PR TITLE
Move deployment code from registry main repo to experimental

### PR DESCRIPTION
**Pull request by shrutiparabgoogle**

### CODE REVIEW

**Review by timburks — APPROVED** [review]

File: deployments/controller/demo/create_apis.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    apg registry create-api-version \
+    --parent projects/demo/locations/global/apis/petstore \
+    --api_version_id $i.0.0 \
+    --api_version.state "PRODUCTION" \
+    --json
+
+    apg registry create-api-spec \
+    --parent projects/demo/locations/global/apis/petstore/versions/$i.0.0 \
+    --api_spec_id openapi.yaml \
+    --api_spec.contents `registry-encode-spec < deployments/controller/demo/data/openapi.yaml@r0` \
```
Comment (giteshk) [inline deployments/controller/demo/create_apis.sh:42]: Is this a tool ? registry-encode-spec

Comment (giteshk) [inline deployments/controller/demo/create_apis.sh:42]: Is this a tool ? registry-encode-spec

**Comment by giteshk** [comment]
Just have one question 

LGTM

**Comment by giteshk** [comment]
Just have one question 

LGTM

**Review by giteshk — APPROVED** [review]

File: deployments/controller/demo/create_apis.sh (hunk 1 above)
Comment (shrutiparabgoogle) [reply deployments/controller/demo/create_apis.sh:42]: Thanks for catching this, it is a tool, but it's in the main repo. This script will fail without that. I can replace that with manual encoding.

Comment (shrutiparabgoogle) [reply deployments/controller/demo/create_apis.sh]: Replaced this tool with manual command: https://github.com/apigee/registry/blob/main/tools/registry-encode-spec/main.go#L28

**Review by giteshk — APPROVED** [review]
LGTM.

I think the registry tool needs an encoding method for Specs or the tool needs to handle encoding of the Specs without having the customer to have to do the manual encoding. 
I will add a ticket for this.

**Pull request merged**